src/content/CloudBoilerplateNet/Services/CachedDeliveryClient.cs

```diff
@@ -30,23 +30,22 @@ public int CacheExpirySeconds
 
         public IContentLinkUrlResolver ContentLinkUrlResolver { get => _client.ContentLinkUrlResolver; set => _client.ContentLinkUrlResolver = value; }
         public ICodeFirstModelProvider CodeFirstModelProvider { get => _client.CodeFirstModelProvider; set => _client.CodeFirstModelProvider = value; }
-        public InlineContentItemsProcessor InlineContentItemsProcessor => _client.InlineContentItemsProcessor;
+        IInlineContentItemsProcessor IDeliveryClient.InlineContentItemsProcessor => _client.InlineContentItemsProcessor;
 
         #endregion
 
         #region "Constructors"
 
         public CachedDeliveryClient(IOptions<ProjectOptions> projectOptions, IMemoryCache memoryCache)
         {
-            if (string.IsNullOrEmpty(projectOptions.Value.KenticoCloudPreviewApiKey))
```

Comment:
you can replace the whole if-else by a single branch with `_client = new DeliveryClient(projectOptions.Value.DeliveryOptions);` because there's a new setting `UsePreviewApi` in the DeliveryOptions to handle this - https://github.com/Kentico/delivery-sdk-net/blob/master/KenticoCloud.Delivery/Configuration/DeliveryOptions%20.cs#L31